marsappspa/src/CounterPage.tsx

```diff
@@ -42,4 +42,4 @@ function Counter() {
     )
 }
 
-export default CounterPage;
```

Comment:
I can't leave comments on the lines above :D 
lines 17- 20
* typo in the variable name
* what about having `JSON.parse(saved) || 0` instead?

JSON.parse(null) should return null - however, this might not work. we could use the conditional operator, though

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator